### PR TITLE
Remove domain models

### PR DESCRIPTION
**Pull request by thyandrecardoso**
Removes the tracking events domain models. Refactors API to accept either a JSON string or a NSDictionary.

### CODE REVIEW

File: Examples/SwiftExample/SwiftExample/ViewController.swift [hunk 1]
```diff
@@ -56,16 +64,26 @@ class ViewController: UIViewController {
 
     @IBAction func sendCustomTrackingEvent(_ sender: Any) {
```
Comment (duartepinto) [inline Examples/SwiftExample/SwiftExample/ViewController.swift:65]: I think we can remove the custom examples, given that they are the same as any other event.

**Review by duartepinto — CHANGES_REQUESTED** [review]
Overall looks good to me, although there are some things that I haven't given a thorough review yet.

Can you update the README and CHANGELOG?

**Comment by thyandrecardoso** [comment]
> Can you update the README and CHANGELOG?

Done in 44a8c2f and fcefb7f. Please take a look.

File: Examples/SwiftExample/SwiftExample/ViewController.swift (hunk 1 above)
Comment (thyandrecardoso) [reply Examples/SwiftExample/SwiftExample/ViewController.swift:65]: I would prefer to leave it. They exhibit a different event type (`custom`) and a different method to use to send a tracking event.

**Review by dimamo5 — APPROVED** [review]

File: .gitattributes [hunk 2]
```diff
@@ -1 +1 @@
-*.pbxproj binary
+*.pbxproj binary merge=union
```
Comment (duartepinto) [inline .gitattributes]: Can you leave a comment here explaining why we are adding this?

**Review by duartepinto — APPROVED** [review]
LGTM!

File: .gitattributes (hunk 2 above)
Comment (thyandrecardoso) [reply .gitattributes]: Done in cf5a11a

**Review by duartepinto — APPROVED** [review]

**Comment by duartepinto** [comment]
We are aware that codecov is failing! The difference is not relevant and we'll merge this anyway

**Pull request merged**